utils/cliutils/commandsflags.go

```diff
@@ -1418,6 +1431,35 @@ var flagsMap = map[string]cli.Flag{
 		Name:  Status,
 		Usage: "[Default: false] Set to true to show the status of the transfer-files command currently in progress.` `",
 	},
+
+	branch: cli.StringFlag{
+		Name:  branch,
+		Usage: "[Optional] Set branch name to filter default value is empty",
+	},
+	pipelineName: cli.StringFlag{
+		Name:  pipelineName,
+		Usage: "[Optional] Set pipeline name default value is empty",
+	},
+	Resources: cli.StringFlag{
+		Name:  Resources,
+		Usage: "[Optional] Set path to the file where resources exist",
+	},
+	name: cli.StringFlag{
+		Name:  name,
+		Usage: "[Optional] Set true to check status for a given branch on every status change",
+	},
+	monitor: cli.BoolFlag{
+		Name:  monitor,
+		Usage: "[Default: false] Set to true to continuously monitor",
+	},
+	repository: cli.StringFlag{
+		Name:  repository,
+		Usage: "[Required] trigger pipeline sync sources repository full name is mandatory",
```

Comment:
It is bad practice to have [Required] flag. Required params should be provided by arguments
```suggestion
		Usage: "[Required] trigger pipeline sync sources repository full name is mandatory",
```

Reply:
Then all the flags are optional?

Reply:
moved required flags as arguments